lib/griddler/email.rb

```diff
@@ -26,6 +27,8 @@ def initialize(params)
       @raw_headers = params[:headers]
 
       @attachments = params[:attachments]
+
+      @vendor_specific = params[:vendor_specific] || {}
```

Comment:
Can this to be `@vendor_specific = params.fetch(:vendor_specific, {})` instead? See the [ruby style guide](https://github.com/bbatsov/ruby-style-guide#hash-fetch-defaults)